### PR TITLE
Use proper "DeletedEntry" type to handle unpublish webhook call

### PR DESCRIPTION
**Pull request by klaari**
Fixes handling unpublish webhook call from Contentful.

Unpublishing wasn't working, the call failed with an error: 
`"Argument 1 passed to Digia\Lumen\ContentfulSync\Http\Controllers\ContentfulSyncController::getEntryContentType() must be an instance of Contentful\Delivery\Resource\Entry, instance of Contentful\Delivery\Resource\DeletedEntry given"`

### CODE REVIEW

File: src/Http/Controllers/ContentfulSyncController.php [hunk 1]
```diff
@@ -92,7 +93,7 @@ public function handleIncomingWebhook(Request $request): Response
             case self::TOPIC_CONTENT_MANAGEMENT_ENTRY_DELETE:
                 /** @var Entry $resource */
```
Comment (Jalle19) [inline src/Http/Controllers/ContentfulSyncController.php:94]: This should probably be DeletedEntry too?

Comment (klaari) [reply src/Http/Controllers/ContentfulSyncController.php]: Good catch! Updated it as well

Comment (klaari) [reply src/Http/Controllers/ContentfulSyncController.php]: @Jalle19 Hello! Had a time to review?

**Comment by Jalle19** [comment]
I'm not really able to maintain this repository anymore so not sure what to do with this PR. @hugovk ?

**Comment by hugovk** [comment]
Hi! I'm in a similar position to @Jalle19 but I think it's okay to merge and release this.

Travis CI had stopped working, I've just replaced it with GitHub Actions: https://github.com/digiaonline/lumen-contentful-sync/pull/24.

@klaari Please could you rebase or merge with `master` so we can check the test still pass? Do you think this needs new tests?

**Comment by klaari** [comment]
Hi @Jalle19 & @hugovk!

Thanks for getting back to me and giving a status update about the repository.
I merged with master, no need for new tests.

**Comment by hugovk** [comment]
Thanks, released as 4.2.0! 🚀

**Pull request merged**